[2022-06-27 13:30:26,463] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-05T00:00:00+00:00 [queued]>
[2022-06-27 13:30:26,559] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-01-05T00:00:00+00:00 [queued]>
[2022-06-27 13:30:26,563] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:26,569] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:30:26,571] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:30:26,959] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-01-05 00:00:00+00:00
[2022-06-27 13:30:27,004] {standard_task_runner.py:52} INFO - Started process 56655 to run task
[2022-06-27 13:30:27,016] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-01-05T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp51mrmbqw', '--error-file', '/tmp/tmpxfu9qget']
[2022-06-27 13:30:27,024] {standard_task_runner.py:80} INFO - Job 29: Subtask model_A
[2022-06-27 13:30:27,338] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-01-05T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:30:27,839] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-01-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-05T00:00:00+00:00
[2022-06-27 13:30:27,848] {python.py:173} INFO - Done. Returned value was: 9
[2022-06-27 13:30:28,049] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220105T000000, start_date=20220627T133026, end_date=20220627T133028
[2022-06-27 13:30:28,199] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:30:28,448] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
